src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -243,7 +244,13 @@ public void setAngleManual(double joystickInput) {
 
 		if (Units.degreesToRotations(getAngle()) > 0.71
 				&& Units.degreesToRotations(getAngle()) < 0.95) {
-			launcherAngleOnePIDController.setReference(manualAngleSetpoint, ControlType.kPosition);
+			launcherAngleOnePIDController.setReference(
+					manualAngleSetpoint,
+					ControlType.kPosition,
+					0,
+					launcherAngleFF.calculate(
+							Units.rotationsToRadians(launcherAngleEncoder.getPosition() - FF_PIVOT_OFFSET),
+							launcherAngleEncoder.getVelocity()));
```

Comment:
Note that the feedforward is based on the setpoint positiion and velocity, so we should be passing in the desired velocity instead of the current velocity, so this should be 0 (same for the other occurrance).

Would also recommend refactoring so the `setReference()` and other logic isn't duplicated.

Reply:
(i know its 15 to midnight BUT does this work?)
![image](https://github.com/robototes/Crescendo2024/assets/145405364/82617ac9-c44c-4c64-a91a-c3967ff38590)

Reply:
Off the top of my head, yes. But best to make the change and push then I can double check.

Reply:
Looking at this again, we should also update this to use the position setpoint instead of the current position from the encoder (this is also mentioned in the docs [here](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/math/controller/ArmFeedforward.html#calculate(double,double))).

Interestingly, last year's version also had this issue (for the wrist, we didn't use `ArmFeedforward` for the arm) - it's possible we were calling `setReference()` often enough that this didn't matter (since the feedforward would've been constantly updated), or that the motors were strong enough to control the (fairly light) wrist without much trouble. Probably best not to rely on either of those things being true this year though.

Reply:
![image](https://github.com/robototes/Crescendo2024/assets/145405364/e94b01e0-4802-4c07-bdcd-237721694832)

would this work?